core/integration/features/spot/orders/0014-ORDT-163.feature

```diff
@@ -0,0 +1,108 @@
+
+    And the opening auction period ends for market "BTC/ETH"
+    When the network moves ahead "1" blocks
+    Then the trading mode should be "TRADING_MODE_CONTINUOUS" for the market "BTC/ETH"
+    And the mark price should be "1000" for the market "BTC/ETH"
+
+  Scenario:0014-ORDT-163, 0014-ORDT-164: A wrapped buy/sell order will be rejected when triggered if the party doesn't have enough of the required quote asset to cover the order.
+    Given the parties place the following orders:
+      | party  | market id | side | volume | price | resulting trades | type       | tif     |
+      | party1 | BTC/ETH   | sell | 10     | 1000  | 0                | TYPE_LIMIT | TIF_GTC |
+      | party2 | BTC/ETH   | buy  | 10     | 1000  | 1                | TYPE_LIMIT | TIF_GTC |
+    # place an order to match with the limit order then check the stop is filled
+    And the parties place the following orders:
+      | party  | market id | side | volume | price | resulting trades | type       | tif     | reference |
+      | party4 | BTC/ETH   | sell | 10     | 1010  | 0                | TYPE_LIMIT | TIF_GTC | p4-sell   |
+
+    # create party1 stop order
+    And the parties place the following orders:
+      | party  | market id | side | volume | price | resulting trades | type       | tif     | only   | ra price trigger | error | reference |
+      | party1 | BTC/ETH   | buy  | 50     | 1010  | 0                | TYPE_LIMIT | TIF_GTC | reduce | 1005             |       | stop1     |
+
+    # now we trade at 1005, this will breach the trigger
+    When the parties place the following orders:
+      | party  | market id | side | volume | price | resulting trades | type       | tif     |
+      | party3 | BTC/ETH   | buy  | 1      | 1005  | 0                | TYPE_LIMIT | TIF_GTC |
+      | party4 | BTC/ETH   | sell | 1      | 1005  | 1                | TYPE_LIMIT | TIF_GTC |
+
+    # check that the order was triggered
+    Then the stop orders should have the following states
+      | party  | market id | status           | reference |
+      | party1 | BTC/ETH   | STATUS_TRIGGERED | stop1     |
+
+    Then "party1" should have general account balance of "200" for asset "ETH"
+    Then "party1" should have general account balance of "1" for asset "BTC"
+    Then "party1" should have holding account balance of "0" for asset "BTC"
+
+    Then the parties amend the following orders:
+      | party  | reference | price | size delta | tif     |
+      | party4 | p4-sell   | 1010  | -10        | TIF_GTC |
+
+    # create party2 stop order
+    And the parties place the following orders:
+      | party  | market id | side | volume | price | resulting trades | type       | tif     | only   | ra price trigger | error | reference |
+      | party2 | BTC/ETH   | sell | 50     | 1015  | 0                | TYPE_LIMIT | TIF_GTC | reduce | 1020             |       | stop2     |
```

Comment:
Again no reduce only for spot market - don't think it matters but good to be clear.